scripts/qa_crowbarsetup.sh

```diff
@@ -4418,6 +4418,14 @@ function oncontroller_testpostupgrade
         scp cirros@$fip:/var/log/ping_outside.out ping_outside.$fip.out
         max=$(sed -n 's/^.* not available for: //p' ping_outside.$fip.out | sort -n | tail -n 1)
         echo "Maximum outage while pinging outside IP from $fip: $max seconds"
+
+        scp cirros@$fip:/mnt/cinder_test.out cinder_test.$fip.out
+        res=$(awk '$1!=p+1{print $1-p}{p=$1}' cinder_test.$fip.out | tail -n +2 | sort | uniq | tail -n 1)
```

Comment:
Im having some difficulty trying to parse this into my head. We are trying to find if there is a gap of more than one second between lines in here right?